src/bus/three_wire.rs

```diff
@@ -0,0 +1,128 @@
+#![allow(clippy::inconsistent_digit_grouping, clippy::unusual_byte_groupings)]
+
+use core::fmt;
+use embedded_hal::blocking::spi::{Transfer, Write};
+
+use crate::bus::Bus;
+
+const WRITE_MODE_MASK: u8 = 0b00000_1_0;
+
+const FIXED_DATA_LENGTH_MODE_1: u8 = 0b000000_01;
+const FIXED_DATA_LENGTH_MODE_2: u8 = 0b000000_10;
+const FIXED_DATA_LENGTH_MODE_4: u8 = 0b000000_11;
+
+// TODO This name is not ideal, should be renamed to FDM
+pub struct ThreeWire<Spi: Transfer<u8> + Write<u8>> {
```

Comment:
What stands FDM for?

Reply:
Fixed length Data Mode (see screenshot of datasheet above).